templates/docker-compose.template

```diff
@@ -37,20 +38,10 @@ services:
       - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
 
   helium-miner:
```

Comment:
Wanted to rename it helium-miner -> hm-gateway. But changing this will required changes to [packet_forwarder](https://github.com/NebraLtd/packet_forwarder) as its configuration files use this as hostname. And then the binary will have to be included in hm-pktfwd.

Considered it low priority for time being. Will open a ticket for it.